bundles/org.eclipse.packagedrone.utils.deb/src/org/eclipse/packagedrone/utils/deb/build/TextFileContentProvider.java

```diff
@@ -26,8 +28,9 @@
 
     public TextFileContentProvider ( final File file ) throws FileNotFoundException, IOException
     {
-        try ( FileReader reader = new FileReader ( file ) )
```

Comment:
Now here we are again at the same situation as above. However I absolutely see the need for a version with an explicit encoding. So I would just add a second constructor with the "encoding" as a parameter.